lendingbot.py

```diff
@@ -181,6 +216,14 @@
 		xDayThreshold = Decimal(config.get("BOT","xdaythreshold"))/100
 		xDays = str(config.get("BOT","xdays"))
 	autorenew = int(config.get("BOT","autorenew"))
+	if(config.has_option('BOT', 'outputCurrency')):
+		outputCurrency = config.get('BOT', 'outputCurrency')
+	if(config.has_option('BOT', 'maxtolent')):
+		maxtolent = config.get('BOT', 'maxtolent')
+	if(config.has_option('BOT', 'maxpercenttolent')):
+		maxpercenttolent = config.get('BOT', 'maxpercenttolent')
+	if(config.has_option('BOT', 'maxtolentrate')):
+		maxtolentrate = config.get('BOT', 'maxtolentrate')
```

Comment:
From lines 222-226, it is incorrectly setting these variables as strings. They must instead be Decimals. They cause errors in lines 354-356 because it is multiplying a string by a number.

ex. `maxtolent = Decimal(config.get('BOT', 'maxtolent'))`